pallets/statistics/src/lib.rs

```diff
@@ -714,6 +812,7 @@ impl<T: Config> Module<T> {
         to_balance: Balance,
         amount: Balance,
         total_supply: Balance,
+        weight_meter: &mut WeightMeter,
     ) -> DispatchResult {
         let asset = AssetScope::Ticker(*ticker);
         let tm = AssetTransferCompliances::<T>::get(&asset);
```

Comment:
Need to make sure the overhead of this function and the for loop below is charged too.

Reply:
I added a benchmark for the loop. The get is already charged since it's before the paused check.